### PR TITLE
Improvements for feature: on policy ma-dpg

### LINKED ISSUE #41: Build ops for computing MLE model learning loss

### LINKED ISSUE #42: Build ops for optimizing the dynamics model parameters via MLE

### PR DESCRIPTION
**Pull request by 0xangelo**
Closes #41 
Closes #42

### CODE REVIEW

File: mapo/agents/mapo/tests/test_dynamics_learning.py [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+@pytest.mark.parametrize("worker_kwargs", get_worker_kwargs())
+def test_model_learns_deterministic_state(worker_kwargs):
+    worker = RolloutWorker(**worker_kwargs)
+    policy = worker.get_policy()
+
+    info = policy.learn_on_batch(worker.sample())
+    initial_dynamics_loss = info[LEARNER_STATS_KEY]["dynamics_loss"]
+    for _ in range(30):
+        new_info = policy.learn_on_batch(worker.sample())
+    assert new_info[LEARNER_STATS_KEY]["dynamics_loss"] <= initial_dynamics_loss
```
Comment (thiagopbueno) [inline mapo/agents/mapo/tests/test_dynamics_learning.py:64]: This test might not pass, since optimizers based on stochastic gradient descent have no guarantees in terms of monotonic improvement in the general case.

I don't think it would be easy to test "learning" even for a MockupEnv in a single step. Maybe if you run `policy.learn_on_batch` until convergence then you should be able to compare with the `initial_dynamics_loss` ..

**Review by thiagopbueno — CHANGES_REQUESTED** [review]

File: mapo/agents/mapo/tests/test_dynamics_learning.py (hunk 1 above)
Comment (0xangelo) [reply mapo/agents/mapo/tests/test_dynamics_learning.py:64]: That would probably take too long. Maybe skip this test for now?

Comment (thiagopbueno) [reply mapo/agents/mapo/tests/test_dynamics_learning.py:64]: Right now you should probably skip it.
